rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -446,7 +446,7 @@ public void registerBtcTransaction(Object[] args) {
         byte[] pmtSerialized = (byte[]) args[2];
         try {
             bridgeSupport.registerBtcTransaction(rskTx, btcTxSerialized, height, pmtSerialized);
-        } catch (IOException | BlockStoreException e) {
+        } catch (IOException | BlockStoreException | RegisterBtcTransactionException e) {
```

Comment:
registerBtcTransaction shouldn't throw RegisterBtcTransactionException. It is caught inside the method

Reply:
Once registerBtcTransaction gets changed to not throw RegisterBtcTransactionException, you should remove the reference of said exception from the tests 😓